config/deploy.rb

```diff
@@ -44,6 +44,9 @@
   'config/property_permissions.yml',
   'config/redis.yml',
   'config/resque-pool.yml',
+  'config/resque-web.rb',
+  'config/initializers/resque_config.rb',
+  'config/initializers/resque_admin.rb',
```

Comment:
What's the rationale behind having Capistrano manage these? We usually do this if there's sensitive information in them, or if the configuration needs to change between servers. Is that the case here?